conf/messages.en

```diff
@@ -97,27 +97,27 @@ typeOfParticipantDetail.error.required = Enter the other type of art market part
 typeOfParticipantDetail.error.length = Other type of art market participant must be 255 characters or less
 typeOfParticipantDetail.error.punctuation = Other type of art market participant can include letters a to z, numbers, spaces, and most punctuation and special characters
 
-boughtOrSoldOverThreshold.title = Has your business made a sale or purchase of art for €10,000 or more after 10 January 2020?
-boughtOrSoldOverThreshold.heading = Has your business made a sale or purchase of art for €10,000 or more after 10 January 2020?
-boughtOrSoldOverThreshold.checkYourAnswersLabel = Has your business made a sale or purchase of art for €10,000 or more after 10 January 2020?
-boughtOrSoldOverThreshold.error.required = Select yes if your business made a sale or purchase of art for €10,000 or more after 10 January 2020
-boughtOrSoldOverThreshold.hint = If you do not use euros, you’ll need to convert the amount to the currency you use.
-
-dateTransactionOverThreshold.title = When was the first sale or purchase of art for €10,000 or more after 10 January 2020?
-dateTransactionOverThreshold.heading = When was the first sale or purchase of art for €10,000 or more after 10 January 2020?
-dateTransactionOverThreshold.checkYourAnswersLabel = When was the first sale or purchase of art for €10,000 or more after 10 January 2020?
-dateTransactionOverThreshold.error.required.all = Enter the date of the first sale or purchase of art for €10,000 or more after 10 January 2020
-dateTransactionOverThreshold.error.required.two = Date of the first sale or purchase must include a {0} and {1}
-dateTransactionOverThreshold.error.required = Date of the first sale or purchase must include a {0}
-dateTransactionOverThreshold.error.invalid = Enter a real date for the first sale or purchase
-dateTransactionOverThreshold.error.startdate = Date of the first sale or purchase must be after 10 January 2020
-dateTransactionOverThreshold.error.future = Date of the first sale or purchase must be in the past
+SoldOverThreshold.title = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.heading = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.checkYourAnswersLabel = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.error.required = Select yes if your business made a sale of art for €10,000 or more after 10 January 2020
+SoldOverThreshold.hint = If you do not use euros, you’ll need to convert the amount to the currency you use.
+
```

Comment:
SoldOverThreshold -> no need for capital S

Reply:
Done